src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -466,18 +463,22 @@ bool AccountStoreSC<MAP>::ParseCreateContractOutput(
       return false;
     }
   } else {
-    std::string outStr{std::istreambuf_iterator<char>(in),
-                       std::istreambuf_iterator<char>()};
+    std::string outStr = {std::istreambuf_iterator<char>(in),
```

Comment:
This seems not fixing the issue. A new `outStr` is still being defined and masks the outter one.

Reply:
Can confirm that this causes failure. I think haichuan has fixed this in some other branch.

Reply:
Sorry it’s a miss. Already fixed